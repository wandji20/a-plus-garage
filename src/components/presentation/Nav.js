/* eslint-disable */
import React from 'react';
import PropTypes from 'prop-types';
import { connect } from 'react-redux';

const Nav = (props) => {
  const { response  } = props;
  // console.log(response);
  return (
    <header className="container bg-info">
      <nav className="nav d-flex justify-content-between">
        <div className="">
          A+ Automobile
        </div>
        <div className="">
          user icon
        </div>
      </nav>
    </header>
  );
};

Nav.propTypes = {
  response: PropTypes.object,
};

Nav.defaultProps = {
  response: {},
};

const mapStateToProps = (state) => ({
  data: state.signUpUser.response
});

export default connect(mapStateToProps)(Nav);
