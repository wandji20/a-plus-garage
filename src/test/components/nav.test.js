import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../../redux/store';
import Nav from '../../components/presentation/Nav';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly', () => {
  const tree = renderer
    .create(<Provider store={store}>
        <BrowserRouter>
          <Nav />
        </BrowserRouter>
      </Provider>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
