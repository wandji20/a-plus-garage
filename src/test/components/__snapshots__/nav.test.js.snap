// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="container d-flex flex-column justify-content-center bg-info remove-padding header"
>
  <nav
    className="nav justify-content-between mx-2"
  >
    <div
      aria-hidden="true"
      className=""
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      <a
        href="/"
        onClick={[Function]}
      >
        <img
          alt="logo"
          src="logo.png"
        />
      </a>
    </div>
    <div
      className=""
    >
      <button
        className="btn"
        onClick={[Function]}
        type="button"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-caret-down fa-w-10 "
          data-icon="caret-down"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 320 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M31.3 192h257.3c17.8 0 26.7 21.5 14.1 34.1L174.1 354.8c-7.8 7.8-20.5 7.8-28.3 0L17.2 226.1C4.6 213.5 13.5 192 31.3 192z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </button>
    </div>
  </nav>
  <div
    className="nav-controls bg-light position-absolute d-flex flex-column align-items-center"
  />
</header>
`;
